.travis.yml

```diff
@@ -1,27 +1,13 @@
 language: python
 
 python:
-    - 2.7
-    - 3.4
     - 3.5
     - 3.6
-    - pypy
+    - 3.7
```

Comment:
Should we also add 3.8?

Reply:
I don't think it's packaged on xenial or bionic, but maybe focal. Let me see what we can do with travis.

Reply:
Cool if it's non-trivial we don't have to block this PR on it. Would be nice to have though.

Reply:
Looks like some tests fail: https://travis-ci.org/github/dropbox/stone/jobs/697871257